tests/cypress/integration/LSP/F116242.spec.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2021 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+
+ * SPDX-License-Identifier: EPL-2.0
+
+ * Contributors:
+ *  Broadcom, Inc. - initial API and implementation
+ */
+
+/// <reference types="Cypress" />
+//@ts-ignore
+/// <reference types="../../support/" />
+
+import { COBOLLS, Theia, VSCODE } from '@eclipse/che-che4z/tests/dist/selectors';
+
+const env = Cypress.env('ide');
+const IDE = env === 'theia' ? Theia : VSCODE;
+
+//F116242 LSP for COBOL:
+context('This is F116242 Spec', () => {
+  beforeEach(() => {
+    cy.updateConfigs('basic');
+  });
+  describe('TC335744 Toggle Enable Commenting out whole section', () => {
+    it(['smoke'], 'Checks Enabling and disabling comment for single line', () => {
+      cy.openFile('ABCD.cbl');
+      cy.getLineByNumber(21).type('{ctrl}/');
+      cy.getLineByNumber(21).contains('*');
```

Comment:
Maybe it will be better to check startsWith instead? Because the '*' must be in the 7th column only.

Reply:
Probably that would make more sense , added a regex for start of line